src/atoms/TextInput/TextInput.stories.js

```diff
@@ -13,6 +13,29 @@ const sizeOptions = {
   auto: 'auto',
 };
 
+const keyboardTypeOptions = {
```

Comment:
Shoud we have some props specifics to native and web? I don't think web has this. Also the `returnKey` types

Reply:
Yeah. Which is why it's grouped separately on the knobs under "Native". This is the best we can do for now to make it clear that these are Native only props.

Reply:
Oh Sooper!!